openstf_dbc/services/model_input.py

```diff
@@ -26,7 +27,7 @@ def get_model_input(
         location="Arnhem",
         datetime_start=None,
         datetime_end=None,
-        forecast_resolution="15T",
+        forecast_resolution="15min",
```

Comment:
Personally I would prefer the pandas convention 15T. Or does pd.DataFrame.resample('15min') now also work?

Reply:
Both are allowed by pandas, I assumed that the 15T is the older convention (but I'm not sure). To me 15min look more clear. 
![image](https://user-images.githubusercontent.com/6715707/132525340-48cfcbf6-7e25-4aab-b935-5c5010dd9d9f.png)
https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#dateoffset-objects